README.md

```diff
@@ -3,14 +3,50 @@
 
 A Go web service that digitally signs device assertion details.
 
-## Install
-Go get it:
+The application can be run in two modes: signing service or admin service. Both the web services
+operate under unencrypted HTTP connections, so these should not be exposed to a public network
+as-is. The services should be protected by web server front-end services, such as Apache, that
+provide secure HTTPS connections. Also, the admin service does not include authentication nor
+authorisation, so this service will typically be made available on a restricted network with some
+authentication front-end on the web server e.g. SSO. Typically, the services will only be available
+on a restricted network at a factory, though, with additional security measures, the signing service
+could be made available on a public network.
+
+Some deployment recommendations are [provided](docs/Deployment.md)
+
+## Install using the snap package
+Are you using one of the many systems that support [snaps](https://snapcraft.io/)?
+A pre-built snap is available from the [Snap store](https://uappexplorer.com/app/serial-vault.james).
+
+```bash
+$ sudo snap install serial-vault
+```
+
+The service needs a PostgreSQL database to run, so:
+- Install PostgreSQL and create a database.
+- Set up the config file, using ```settings.yaml``` as a guide.
+- Configure the snap using the ```settings.yaml``` file:
+
+```bash
+$ cat /path/to/settings.yaml | sudo /snap/bin/serial-vault.config
+$ sudo systemctl restart snap.serial-vault.serial-vault.service
+```
+
+The snap will create the tables in the database on restart, as soon as it has a valid database connection.
+
+The service mode (signing or admin) is defined in the settings.yaml file. The
+selected service should be accessible on port :8080 or :8081:
+ - Signing Service: http://localhost:8080/v1/version
+ - Admin Service: http://localhost:8081/
+
+## Install from Source
+If you have a Go development environment set up, Go get it:
 
   ```bash
   $ go get github.com/ubuntu-core/identity-vault
   ```
 
-Configure it:
+### Configure it:
 - Install PostgreSQL and create a database.
 - Set up the config file, using ```settings.yaml``` as a guide.
```

Comment:
I've not been explicit here because configuring PostgreSQL is a bit more of a complex area, same with the Apache configuration. There are many ways to set things up, and taking one approach in development may be okay but may not be advisable in production. My concern is that in trying to provide detailed step-by-step instructions, we may lead deployers down the wrong path - especially if they start using dev-oriented instructions in production.

I've placed more of an emphasis on the Juju bundle and the PostgreSQL and Serial Vault charms handle the database creation and configuration themselves.